ci/scripts/github/checks.sh

```diff
@@ -20,31 +20,44 @@ source ${WORKSPACE}/ci/scripts/github/common.sh
 
 update_conda_env
 
-fetch_base_branch
+log_toolchain
 
-rapids-logger "Checking copyright headers"
-python ${MORPHEUS_ROOT}/ci/scripts/copyright.py --verify-apache-v2 --git-diff-commits ${CHANGE_TARGET} ${GIT_COMMIT}
+cd ${MORPHEUS_ROOT}
 
-rapids-logger "Runing Python style checks"
-${MORPHEUS_ROOT}/ci/scripts/python_checks.sh
+fetch_base_branch
 
 git submodule update --init --recursive
 
 rapids-logger "Configuring cmake for Morpheus"
 CMAKE_FLAGS="${CMAKE_BUILD_ALL_FEATURES}"
+CMAKE_FLAGS="${CMAKE_FLAGS} -DMORPHEUS_PYTHON_BUILD_STUBS=OFF"
+export CMAKE_FLAGS="${CMAKE_FLAGS} -DMORPHEUS_PYTHON_INPLACE_BUILD=ON"
 if [[ "${LOCAL_CI}" == "" ]]; then
     CMAKE_FLAGS="${CMAKE_FLAGS} -DCCACHE_PROGRAM_PATH=$(which sccache)"
 fi
+
 cmake -B build -G Ninja ${CMAKE_FLAGS} .
 
-rapids-logger "Building targets that generate source code"
-cmake --build build --target morpheus_style_checks --parallel ${PARALLEL_LEVEL}
+rapids-logger "Building Morpheus"
+cmake --build build --parallel ${PARALLEL_LEVEL}
```

Comment:
Why are we doing the full build now?

Reply:
This is needed so that pylint can accurately perform the `no-member` check which previously we had to disable.

Reply:
Ahhh yes. Forgot about that